src/app/pages/client/explore/Server.tsx

```diff
@@ -512,7 +533,77 @@ export function PublicRooms() {
                 {server}
               </Text>
             </Box>
-            <Box grow="Yes" basis="No" />
+            <Box
+              shrink="No"
+              grow={screenSize === ScreenSize.Mobile ? 'No' : 'Yes'}
+              basis={screenSize === ScreenSize.Mobile ? 'Yes' : 'No'}
+              justifyContent="End"
+            >
```

Comment:
it looks like the server name doesn't align properly in center on mobile screen size